src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const largeIMG = document.getElementById('largeImg');
+
+const smallIMG = document.getElementById('thumbs');
+
+smallIMG.addEventListener('click', (evnt) => {
+  evnt.preventDefault();
```

Comment:
Consider whether `evnt.preventDefault()` is necessary. If the links are not meant to navigate to another page, then this is fine. However, if they are supposed to navigate, you might want to remove this line or conditionally prevent the default action only when necessary.